go/chat/storage_test.go

```diff
@@ -0,0 +1,255 @@
+package chat
+
+import (
+	"bytes"
+	"crypto/rand"
+	"encoding/binary"
+	"testing"
+
+	"github.com/keybase/client/go/chat/pager"
+	"github.com/keybase/client/go/externals"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/chat1"
+	"github.com/keybase/client/go/protocol/gregor1"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func setupStorageTest(t *testing.T, name string) (libkb.TestContext, *Storage) {
+	tc := externals.SetupTest(t, name, 2)
+	return tc, NewStorage(tc.G)
+}
+
+func randBytes(n int) []byte {
```

Comment:
Yeah they are everywhere, I never know where, so I usually just ends up copying it.